src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -135,6 +156,10 @@ public static void assertCommandSuccess(Command command, Model actualModel, Stri
         assertCommandSuccess(command, actualModel, expectedCommandResult, expectedModel);
     }
 
+    public static void asserCommandSuccess() {
```

Comment:
is this method necessary, because its just an empty method? also the method name has spelling error